### PR TITLE
Changes the placement of a callout

### PR DESCRIPTION
**Pull request by xenolinux**
This PR fixes the warning[1] given by the `asciibinder build` command for enterprise-4.6 branch.

[1] `asciidoctor: WARNING: modules/graceful-shutdown.adoc: line 47: no callout found for <1>
     asciidoctor: WARNING: modules/graceful-shutdown.adoc: line 60: no callout found for <1>`

Version(s): 4.6

Issue: None

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/ascii-warning/backup_and_restore/graceful-cluster-shutdown.html

Additional information:

QE: @sunilcio

### CODE REVIEW

File: modules/graceful-shutdown.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 <1>; done
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 ; done <1>
```
Comment (pneedle-rh) [inline modules/graceful-shutdown.adoc:45]: @xenolinux hi! Thank you for addressing this issue.

You could change `1 ;` to `1;`. Removing the space before the semi-colon is consistent with the example earlier in the command. Thanks!

Comment (xenolinux) [reply modules/graceful-shutdown.adoc]: @pneedle-rh Addressed

File: modules/graceful-shutdown.adoc [hunk 2]
```diff
@@ -42,7 +42,7 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 <1>; done
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done <1>
 ----
 <1> Indicates how long, in minutes, this process lasts before the control-plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to 10 minutes or longer to make sure all the compute nodes have time to shut down first.
```
Comment (pneedle-rh) [inline modules/graceful-shutdown.adoc:47]: I just reviewed this line in the later releases and I recall that "Indicates how long" was updated to "`-h 1` indicates how long" when the call out numbering was moved for line 45. You could update this line in this PR for 4.6 too.

[Here](https://github.com/openshift/openshift-docs/blame/main/modules/graceful-shutdown.adoc#L47) is an example of the updated call out in another branch, in this case main.

Comment (xenolinux) [reply modules/graceful-shutdown.adoc]: Addressed

**Comment by pneedle-rh** [comment]
LGTM!

**Pull request merged**